packages/flutter_tools/test/src/schema.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:file/file.dart';
```

Comment:
Maybe name this pubspec_schema to be a little more specific?

Reply:
Done!